calibrations/mbd/MBDTRKZ/prorun_mbdtrkz.sh

```
@@ -0,0 +1,124 @@
+      else [[ $# -lt 2 ]] && usage; listdir=$2; shift 2; fi ;;
+    --nevtperjob|--nevtperjob=*)
+      if [[ "$1" == *=* ]]; then list_nevents="${1#*=}"; shift
+      else [[ $# -lt 2 ]] && usage; list_nevents=$2; shift 2; fi ;;
+    -*)    usage ;;
+    *)     [[ -n "$runnumber" ]] && usage; runnumber=$1; shift ;;
+  esac
+done
+
+[[ -z "$runnumber" ]] && usage
+
+subdir="${ORIG_DIR}/MBDTRKZ/${runnumber}/$(printf '%04d' ${trkseg})"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)